apps/i18n/common/en_us.json

```diff
@@ -354,10 +354,14 @@
   "courses": "Courses",
   "coursesCardAction": "Find a course",
   "coursesCardDescription": "Find a new course to keep challenging yourself.",
-  "coursesHeadingDescription": "Create an account to save your progress and projects. Or just start coding - no account needed. All courses are available at no cost.",
-  "coursesHeadingStudent": "Learn Computer Science",
-  "coursesHeadingSubText": "{linesCount} lines of code written by {studentsCount} million students.",
-  "coursesHeadingTeacher": "Teach Computer Science",
+  "coursesLearnHeroButton": "Get started",
+  "coursesLearnHeroDescription": "Get started coding today. Our courses and activities are free! It’s easier - and way more fun - than you ever thought. Create an account to save your projects.",
+  "coursesLearnHeroHeading": "Anyone can learn computer science",
+  "coursesLearnHeroSubHeading": "{studentsCount} million students have learned on Code.org!",
+  "coursesTeachHeroButton": "Get started",
+  "coursesTeachHeroDescription": "More than one million teachers have brought CS to their students using Code.org. We make it easy, no matter your background. ",
+  "coursesTeachHeroHeading": "You Can Teach Computer Science",
+  "coursesTeachHeroSubHeading": "You don't have to be a software developer to teach computer science.",
```

Comment:
Changed the i18n keys to something I found more intuitive; I assume that's okay, since we'll need to get these translated anyway.